README.md

```diff
@@ -102,9 +102,12 @@ $result->getErrorMessages();
 
 ## Documentation
 
-- [Guide](docs/guide/en/README.md)
```

Comment:
Russian is not complete. It early for add link in readme,

Reply:
I have already translated it into Portuguese. If possible, do not remove from readme.md and the folder